### ISSUE TITLE
anunāsikas incorrectly turned into anusvāras

### ISSUE THREAD

**Issue by sumanthegde** [sanskrit-lexicon/csl-ldev#17]
Example: https://github.com/sanskrit-lexicon/csl-ldev/blob/main/v02/ap90/10692.txt
कान्तिः has become कांतिः here.
कांतिः is incorrect as per my limited knowledge.

**Comment by sumanthegde** [comment]
The origin of this is perhaps a certain edition of apte dictionary which made this change for printing convenience.
![apte_anuswara_explanation](https://user-images.githubusercontent.com/2841785/219544390-1fa48e58-bdcb-4f24-833a-44fffa38ee52.jpg)

**Comment by gasyoun** [comment]
>made this change for printing convenience.

@drdhaval2785 can we say it's a Hindi typograhic convention?

**Comment by drdhaval2785** [comment]
Not really. It seems that the rule https://ashtadhyayi.com/sutraani/8/4/58 is mandatory and not optional. So, it seems to be a typing convention rather than grammatical convention.

**Comment by gasyoun** [comment]
<details>
<summary>🤖 Bug triage findings</summary>

**Bug class:** source-error (systematic encoding convention — anusvāra M used instead of anunāsika n in SLP1)

**Evidence:**

- `csl-orig/v02/ap90/ap90.txt` line 91630: `<k1>kAMtiH<k2>kAMtiH` — the headword is encoded as `kAMtiH` (SLP1 anusvāra) throughout
- `csl-devanagari/v02/ap90/ap90.txt` line 89735: `<k1>कांतिः<k2>कांतिः` — renders the same anusvāra form
- `csl-ldev/v02/ap90/10692.txt` faithfully reflects both upstreams (no bug in ldev itself)
- Pattern `kAMti` appears **59 times** in `ap90.txt` — this is a systematic encoding choice, not an isolated typo

**Agent-fixable (Tier A)?** No — needs an editorial decision from project maintainers

The fix is technically straightforward: replace `kAMti` → `kAnti` in `csl-orig/v02/ap90/ap90.txt` (59 occurrences) and regenerate downstream files. But the 2023 discussion here noted that some editions of the Apte dictionary itself used the anusvāra form for printing convenience, and drdhaval2785 cited Ashtadhyayi 8.4.58 noting the rule is mandatory (not optional). This makes it a **project-level editorial decision** before any code change.

**If Tier D — what needs checking:** N/A

**Suggested next step:** Project maintainers decide: does AP90 encoding policy follow the printed Apte edition (keep `kAMtiH`) or the strict SLP1 convention for Sanskrit-as-written (change to `kAntiH`)? Once decided, a batch replace on csl-orig ap90.txt covers all 59 occurrences in one pass.

</details>
